app/scripts/views/sign_up.js

```diff
@@ -183,6 +183,8 @@ define(function (require, exports, module) {
         signinUri: this.broker.transformLink('/signin')
       };
 
+      this.setShowSyncSuggestion(context);
```

Comment:
instead of doing this can you try just using the mixin roughly like so:

```diff
diff --git a/app/scripts/views/mixins/signin-mixin.js b/app/scripts/views/mixins/signin-mixin.js
index 4da1d95..26cc1e4 100644
--- a/app/scripts/views/mixins/signin-mixin.js
+++ b/app/scripts/views/mixins/signin-mixin.js
@@ -111,10 +111,20 @@ define(function (require, exports, module) {
     */
 
     setShowSyncSuggestion: function (context) {
+
+    },
+
+    context: function () {
+      var context = {
+
+      }
+
       if (! this.relier.get('service') && ! context.isSyncMigration && ! context.isAmoMigration) {
         context.showSyncSuggestion = true;
         this.metrics.logEvent('sync-suggest.visible');
       }
+
+      return context
     },
 
     /**
diff --git a/app/scripts/views/sign_in.js b/app/scripts/views/sign_in.js
index 75453dc..f958f75 100644
--- a/app/scripts/views/sign_in.js
+++ b/app/scripts/views/sign_in.js
@@ -89,7 +89,7 @@ define(function (require, exports, module) {
         suggestedAccount: hasSuggestedAccount
       };
 
-      this.setShowSyncSuggestion(context);
+      //this.setShowSyncSuggestion(context);
       return context;
     },
 
diff --git a/app/scripts/views/sign_up.js b/app/scripts/views/sign_up.js
index 06ea841..4fa34ca 100644
--- a/app/scripts/views/sign_up.js
+++ b/app/scripts/views/sign_up.js
@@ -183,7 +183,7 @@ define(function (require, exports, module) {
         signinUri: this.broker.transformLink('/signin')
       };
 
-      this.setShowSyncSuggestion(context);
+      //this.setShowSyncSuggestion(context);
 
       return context;
     },

```